modules/tinymce/src/core/test/ts/browser/focus/FocusControllerTest.ts

```diff
@@ -118,5 +135,23 @@ describe('browser.tinymce.core.focus.FocusControllerTest', () => {
       editor.getBody().focus();
       assertEditorBodyInsideViewport(editor);
     });
+
+    it('TINY-12626: Editor is inside the viewport when focused, regardless if it stars above or below the viewport', async () => {
+      const editor = hook.editor();
+
+      Scroll.to(0, 0, SugarDocument.getDocument());
+      assertEditorBodyIsOutsideViewport(editor);
+      editor.getBody().focus();
+      assertEditorBodyInsideViewport(editor);
+      editor.getBody().blur();
+      Scroll.to(0, 1000, SugarDocument.getDocument());
+      assertEditorBodyIsOutsideViewport(editor);
```

Comment:
Could we split this assert into two separate functions so that it's above and below it would make this easier to understand.
Don't we also need a editor that is higher than the scroll viewport to test with? So maybe add a separate context with a separate editor hook and wrap the exiting ones in context as a "small editor".

We want to test if editor is above then scroll into view, if the editor is below then scroll into view, if editor is already in view don't scroll. That test should pass for both a editor smaller than the viewport and higher than the viewport.